static/src/js/components/PublishPreview/PublishPreview.jsx

```diff
@@ -74,8 +80,8 @@ const PublishPreview = () => {
         // The correct version of the metadata has been fetched
         setPreviewMetadata(fetchedMetadata)
```

Comment:
feel like the names are confusing me a little.   Can we call this fetchedPreviewMetadata?